libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -859,7 +859,7 @@ void AP_TECS::_update_throttle_without_airspeed(int16_t throttle_nudge)
     _pitch_demand_lpf.apply(_pitch_dem, _DT);
     const float pitch_demand_hpf = _pitch_dem - _pitch_demand_lpf.get();
     _pitch_measured_lpf.apply(_ahrs.get_pitch(), _DT);
-    const float pitch_corrected_lpf = _pitch_measured_lpf.get() - radians(0.01f * (float)aparm.pitch_trim_cd);
+    const float pitch_corrected_lpf = _pitch_measured_lpf.get();
```

Comment:
I wonder if we should add a comment here that it might be a good idea to include pitch trim. I don't think we worked if we should remove or fix.

Reply:
Or break this bit out in to another PR and we can get Paul to take a look.

Reply:
the variable disappears so can't really break it out, but I can ask @priseborough to decide if we should be subtracting pitch trim here

Reply:
You don't have to remove the variable in this PR, its not linked to the parameter, which is why it was not working.